src/components/controller/loader.ts

```diff
@@ -0,0 +1,58 @@
+import { ArticlesType, SourcesType } from "../interfaces";
+
+class Loader {
+  baseLink: string;
+  options: object;
+
+  constructor(baseLink: string, options: object) {
```

Comment:
Can we typify options another way or is it impossible?

Reply:
yes we can  add public to params